Sources/EastAsianWidth/String+Extension.swift

```diff
@@ -9,45 +9,52 @@ import Foundation
 
 public extension String {
 
-    public var containsEastAsianAmbiguousCharacters: Bool {
+    var containsEastAsianAmbiguousCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianAmbiguous }
     }
 
-    public var containsEastAsianHalfwidthCharacters: Bool {
+    var containsEastAsianHalfwidthCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianHalfwidth }
     }
 
-    public var containsEastAsianFullwidthCharacters: Bool {
+    var containsEastAsianFullwidthCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianFullwidth }
     }
 
-    public var containsEastAsianNarrowCharacters: Bool {
+    var containsEastAsianNarrowCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianNarrow }
     }
 
-    public var containsEastAsianNeutralCharacters: Bool {
+    var containsEastAsianNeutralCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianNeutral }
     }
 
-    public var containsEastAsianWideCharacters: Bool {
+    var containsEastAsianWideCharacters: Bool {
         return unicodeScalars.contains { $0.isEastAsianWide }
     }
 
     // MARK: -
 
-    public var containsFullwidthOrAmbiguousCharacters: Bool {
+    var containsFullwidthOrAmbiguousCharacters: Bool {
         return unicodeScalars.contains { $0.isFullwidthOrAmbiguous }
     }
 
-    public var containsFullwidthCharacters: Bool {
+    var containsFullwidthCharacters: Bool {
         return unicodeScalars.contains { $0.isFullwidth }
     }
 
-    public var containsHalfwidthOrAmbiguousCharacters: Bool {
+    var containsHalfwidthOrAmbiguousCharacters: Bool {
         return unicodeScalars.contains { $0.isHalfwidthOrAmbiguous }
     }
 
-    public var containsHalfwidthCharacters: Bool {
+    var containsHalfwidthCharacters: Bool {
         return unicodeScalars.contains { $0.isHalfwidth }
     }
+
+    /// Characters count up by O(n).
+    /// If the character is ASCII code then count is 1, or else count is 2.
+    /// Return value is sum of characters count.
+    var fullwidthHalfwidthCount: Int {
+        return unicodeScalars.countByEastAsianWidth()
+    }
```

Comment:
This repo is mainly for only supporting around Unicode things, but this line is specific for Japanese things.  I think we don't need to support it by this project and library use can define it by themselves

Reply:
```
public extension UnicodeScalar {
    var isEastAsianFullwidth: Bool {
        switch self.value {
        case 0x3000...0x3000: return true
        case 0xff01...0xff60: return true
        case 0xffe0...0xffe6: return true
        default: return false
        }
    }
```
You see this function. EastAsianWidth is a part of unicode, it's OK to include it